lib/web_ui/lib/src/engine/compositor/initialization.dart

```diff
@@ -51,5 +51,10 @@ js.JsObject canvasKit;
 /// The Skia font collection.
 SkiaFontCollection skiaFontCollection;
 
+/// Initializes [skiaFontCollection].
+void ensureSkiaFontCollectionInitialized() {
```

Comment:
Once this file is opted in, can this become `late SkiaFontCollection = skiaFontCollection`?  Maybe final if it doesn't get reset?

Reply:
Hopefully. Filed https://github.com/flutter/flutter/issues/59371 to revisit this.